src/energyplus/GeometryTranslator.hpp

```diff
@@ -66,8 +66,7 @@ namespace energyplus {
    *  \relates GeometryTranslator */
   OPENSTUDIO_ENUM(CoordinateSystem,
     ((Relative))
-    ((World))
-    ((Absolute))
+    ((Absolute)(World))
```

Comment:
Simplify enum, since 'Absolute' and 'World' mean the same thing. This will avoid problems like forgetting to do it `if World || if Absolute`